libs/vertexai/langchain_google_vertexai/chat_models.py

```diff
@@ -938,8 +812,10 @@ class AnswerWithJustification(BaseModel):
 
     def bind_tools(
         self,
-        tools: Sequence[Union[Type[BaseModel], Callable, BaseTool]],
-        tool_config: Optional[Dict[str, Any]] = None,
+        tools: Sequence[Union[_FunctionDeclarationLike, VertexTool]],
+        tool_config: Optional[_ToolConfigDict] = None,
```

Comment:
nits: should we add to the docstring an example how to contsruct _ToolConfigDict and _ToolChoiceType?

so that people don't need to read the source code.